index.d.ts

```diff
@@ -23,7 +23,7 @@ declare namespace File {
      *
      * Default: `undefined`
      */
-    path?: string;
+    path?: string | null | undefined;
```

Comment:
Ok, I did not remember the rule for interfaces (for functions `?` is `| undefined` but I wasn't sure). I'll update it.